.github/CONTRIBUTING.md

```diff
@@ -1,38 +1,37 @@
-Contributing to FOSSProfiles
-============================
+# Contributing to FOSSProfiles
```

Comment:
My markdown linter does not like different styles of heading in the same file. I think this helps with consistency anyway